### ISSUE TITLE
Road Segments not showing

### ISSUE THREAD

**Issue by SirTificate** [openbikesensor/portal#195]
In the latest version is a switch to enable road segments on the map, but the segments aren't showing.
Are these already implemented or is it work-in-progress?

**Comment by gluap** [comment]
For me it looks like it works - How does it look for your / What are your expectations for the visualisation of road segments? If there are no road segments at all, you may have to look into importing (or re-importing) the openstreetmap ``.shp`` file of the respective area.

These are screenshots of my test instance, with (in this order): Segments off, segments on, segments on and one segment clicked:
![image](https://user-images.githubusercontent.com/44007906/153279305-0e13e809-9ff6-43e7-9acd-d720bde115fa.png)
![image](https://user-images.githubusercontent.com/44007906/153279363-14428b50-be1c-4391-877e-aaab827fd42b.png)
 ![image](https://user-images.githubusercontent.com/44007906/153279075-854801c1-e6af-4eb8-bd86-3152ca55a18a.png)

**Comment by SirTificate** [comment]
I expected to see colored roads, but all I can see are the event points as shown in the screenshot.
<img width="1501" alt="Bildschirmfoto 2022-02-09 um 21 41 12" src="https://user-images.githubusercontent.com/36823811/153286335-e7123c06-cbbf-44cd-81bf-a1e4120b1055.png">

It is probably also worth mentioning, that there's somewith wrong with the event points as well.
When I click an event point a small notification appears which says "no road found".

<img width="645" alt="Bildschirmfoto 2022-02-09 um 21 43 49" src="https://user-images.githubusercontent.com/36823811/153286689-2adb6a55-7219-43fb-8292-fdc3f567e097.png">
 
Unfortunately I have to idea how to debug this issue atm. I could look up anything if I had a hint what I need to look for.

**Comment by SirTificate** [comment]
The request to 
`/api/mapdetails/road?longitude=10.095478334208783&latitude=53.639888649364266&radius=100` returns an empty json object. Does this help in any way?

**Comment by SirTificate** [comment]
I hate spamming, but I figured it myself.
A on Monday I reset the database but did not reimport the OSM data again. Hence the processing of the uploaded tracks didn't return any valuable data besides the confirmed event points.
I set one track's processing_status to queued, watched the worker process reprocessing the track ét voila.. there it is! So, as I'm writing this update the worker is processing all previously updated tracks again and the map is finally showing colored road segments.
Thanks for looking into this. I'm gonna close this ticket now. 👍
